crates/esthri/src/lib.rs

```diff
@@ -328,3 +319,77 @@ where
 
     Ok(listing)
 }
+
+pub mod opts {
+    use derive_builder::Builder;
+    use glob::Pattern;
+
+    use crate::rusoto::*;
+    #[derive(Debug, Copy, Clone, Builder)]
+    pub struct AwsCopyOptParams {
```

Comment:
Should this be `EsthriCopyOptParams` for consistency?

Reply:
This is for the aws compat mode, the esthri mode doesn't have the copy option so I thought I make a distinction here